docs/manifest/reportingoptions.md

```diff
@@ -0,0 +1,53 @@
+---
+title: ReportingOptions element in the manifest file (preview)
+description: The ReportingOptions element specifies the reporting options listed in the pre-processing dialog of a spam reporting add-in in Outlook..
+ms.date: 06/22/2023
+ms.localizationpriority: medium
+---
+
+# ReportingOptions element (preview)
+
+Specifies the reporting options listed in the pre-processing dialog of a spam reporting add-in in Outlook.
+
+**Add-in type**: Mail
+
+**Valid only in these VersionOverrides schemas**:
+
+- Mail 1.1
+
+For more information, see [Version overrides in the manifest](/office/dev/add-ins/develop/add-in-manifests#version-overrides-in-the-manifest).
+
+**Associated with these requirement sets**:
+
+- [Mailbox preview](../requirement-sets/outlook/preview-requirement-set/outlook-requirement-set-preview.md)
+
+## Contained in
+
+- [PreProcessingDialog (preview)](preprocessingdialog.md)
+
+## Attributes
+
+None.
+
+## Child elements
+
+| Element | Required | Description |
+| :------ | :------: | :------ |
+| **Title** | Yes | Specifies the custom title of the pre-processing dialog. Its **resid** attribute must be set to the value of the **id** attribute of a [String](string.md) in the [ShortStrings](shortstrings.md) element under the [Resources](resources.md) element. |
+| **Option** | Yes | Specifies a custom option that a user can select from the pre-processing dialog to provide a reason for reporting a message. You can add up to five options, but must set at least one option. |
```

Comment:
Is it possible to put more emphasis on the word **five** I don't know if this is standard or normal. 

I just feel like we are going to get github questions about "This manifest doesn't work. They send the manifest...and it has like 6 options in it :)

Reply:
Per the Microsoft style guide, we can italicize text to emphasize it, so I went ahead and did that based on your feedback.